src/views/Tag/Tags.vue

```diff
@@ -30,6 +30,49 @@
               >#f9f9f9</md-chip
             >
           </md-table-cell>
+          <md-table-cell md-label="Subscriber">
+            <div
+              v-if="item.subscriber !== null && item.subscriber.length !== 0"
+              class="flex-start"
+            >
+              <div
+                v-for="subscriber in item.subscriber"
+                :key="item.subscriber.indexOf(subscriber)"
+                :set="(subscriberDetail = getSubscriberDetail(subscriber))"
+                
+              >
+                <img
+                  v-if="subscriberDetail.profileImage"
+                  class="profile-image"
+                  :src="subscriberDetail.profileImage"
+                />
+                <img
+                  v-else
+                  class="profile-image"
+                  src="@/assets/images/account_placeholder.png"
+                />
+              </div>
+            </div>
+          </md-table-cell>
+          <md-table-cell md-numeric>
+            <md-menu md-size="small" md-align-trigger>
+              <md-button md-menu-trigger class="md-icon-button">
+                <md-icon> more_horiz </md-icon>
+              </md-button>
+
+              <md-menu-content>
+                <md-menu-item
+                  v-if="!isUserSubscribed(firebaseUser, item.subscriber)"
+                  @click="subscribe(item)"
+                >
+                  Subscribe
+                </md-menu-item>
+                <md-menu-item v-else @click="unsubscribe(item)">
+                  Unsubscribe
+                </md-menu-item>
+              </md-menu-content>
```

Comment:
Actually, This page will display only the tag that own by the user or user has subscribed to. So If you create the action button for each tags in table, User won't have a chance to subscribe the other tag